code/modules/reagents/reagent_containers/bottle.dm

```diff
@@ -382,7 +382,7 @@
 /obj/item/reagent_containers/glass/bottle/necropolis_seed
 	name = "bowl of blood"
 	label_name = "blood"
-	desc = "A clay bowl containing a fledgling Necropolis, preserved in blood. A robust virologist may be able to unlock its full potential..."
+	desc = "A clay bowl containing a fledgling Necropolis, preserved in blood. A robust member of medical staff may be able to unlock its full potential..."
```

Comment:
Even if the job doesn't technically exist, the role does and there is simply no better way to describe it.
```suggestion
	desc = "A clay bowl containing a fledgling Necropolis, preserved in blood. A robust virologist may be able to unlock its full potential..."
```

Reply:
I'm thinking of future players that may not know that virologist used to be a proper job, and wonder who the "virologist" is when coming to the station with that loot. In my opinion, it's best to just vaguely point them in the direction of medbay with that item.